src/casm/clex/ConfigIOStrucScore.cc

```diff
@@ -117,6 +118,8 @@ bool StrucScore::init(const Configuration &_tmplt) const {
 
   m_strucmapper = notstd::make_unique<StrucMapper>(
       PrimStrucMapCalculator(*m_altprim), m_strain_weight);
+
+  for (Index i=0; i < m_prop_names.size(); i++) _add_rule(std::vector<Index>({i}));
```

Comment:
```suggestion
  for (Index i=0; i < m_prop_names.size(); i++) {
      _add_rule(std::vector<Index>({i}));
  }
```
readability

Reply:
Really, though, this seems like a regression in `Base1DDatumFormatter`. I think that the default behavior of this DatumFormatter is that all entries should be output if the index rules aren't specified.